server/lib/data_import/my_anime_list/extractor/character.rb

```diff
@@ -0,0 +1,107 @@
+module DataImport
+  class MyAnimeList
+    module Extractor
+      class Character
+        include Enumerable
+
+        attr_reader :dom, :external_id
+
+        def initialize(html, external_id)
+          @dom = Nokogiri::HTML(html)
+          @external_id = external_id
+        end
+
+        def name
+          # line 473 in guts-character.html
+          options = ['Animeography', 'Mangaography', 'Voice Actors']
+          dom.css('.normal_header').map do |row|
```

Comment:
You can probably do `dom.css('#horiznav_nav ~ .normal_header').first` to get this instead, no `map` needed.

Also you probably didn't mean `map`.  `map` is for turning a list of `A, B, C, D, E, F` into a list `A1, B1, C1, D1, E1, F1`.  It transforms each item in a list